inc/item_devices.class.php

```diff
@@ -475,17 +475,17 @@ static function getDeviceForeignKey() {
     * @param $item
     * @param $table
     * @param $options            array
-    * @param $delete_all_column          (default NULL)
+    * @param $delete_all_column
     * @param $common_column
     * @param $specific_column
-    * @param $delete_column               (default NULL)
+    * @param $delete_column
     * @param $dynamic_column
    **/
    function getTableGroup(CommonDBTM $item, HTMLTableMain $table, array $options,
-                          HTMLTableSuperHeader $delete_all_column=NULL,
+                          HTMLTableSuperHeader $delete_all_column,
```

Comment:
Seems OK since there are only 2 calls to this method; in the same class, and all is properly defined on that side.